F1T.sln

```diff
@@ -4,7 +4,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 17.0.32112.339
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "F1T", "F1T\F1T.csproj", "{95FACA9F-4C85-40AB-A672-B5E508E37161}"
-EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "F1TMock", "F1TMock\F1TMock.csproj", "{981B8F8C-2E07-4A4B-877B-EBD1738F069D}"
```

Comment:
@ryanabaker I have to test this later. These two project lines have the same ID at the start

Reply:
im confusion, but yes

Reply:
VS auto changed it when opening